tutorials/hamiltonian_simulation/hamiltonian_simulation_guide.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.            num_qdrift=30,
</pre><p>why did you choose 30? is it according to the error formula you have written above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>